examples/spreadsheet.md

```diff
@@ -140,12 +140,52 @@ function compile(input) {
 ## User Interaction
 
 ```javascript
+
+const SELECTED_POSITION = {x: 0, y: 0};
+```
+
+We will need a way to track the `SELECTED_POSITION` in the `regular-table` 
+with the `x` and `y` coordinates currently focused so that we can scroll 
+to another distant part of the table and back with our selection preserved.
+We can default it to the origin.
+
```

Comment:
Please leave out as much leading/trailing white-space as you can in the future for code segments - this is stripped via processing script for bl.ocks but it is still a bit cumbersome to read.